TransmitBias/scripts/AFI_IndividualAlignRawAndInitialMaps.sh

```diff
@@ -169,7 +169,7 @@ else
 	    --bbregister-regfile-out="$WorkingDirectory"/AFI_orig_bbregister.dat
 fi
 
-if [[ "$GradientDistortionCoeffs" == "" ]]
+if [[ "$GradientDistortionCoeffs" == "" && "$IsLongitudinal" == "0" ]] || [[ -f "$AFIFolder/AFI_orig_gdc_warp.nii.gz" && "$IsLongitudinal" == "1" ]]
```

Comment:
This appears to say "in longitudinal mode, if the warpfield exists, don't use it (and if it doesn't exist, do attempt to use it)".
```suggestion
if [[ "$GradientDistortionCoeffs" == "" && "$IsLongitudinal" == "0" ]] || [[ ! -f "$AFIFolder/AFI_orig_gdc_warp.nii.gz" && "$IsLongitudinal" == "1" ]]
```

Reply:
The intention is: "if gradient coeffs do not exist, don't use them" (then-clause), otherwise do use them (else clause)
In longitudinal mode, gradient coeffs are not checked, but the gdc_warp field file is checked instead, and the logic stays the same.